01-node-tutorial/11-fs-async.js

```diff
@@ -1,29 +1,29 @@
-const { readFile, writeFile } = require('fs')
+const { readFile, writeFile } = require("fs");
 
-console.log('start')
-readFile('./content/first.txt', 'utf8', (err, result) => {
+console.log("start");
+readFile("./content/first.txt", "utf8", (err, result) => {
   if (err) {
-    console.log(err)
-    return
+    console.log(err);
+    return;
   }
-  const first = result
-  readFile('./content/second.txt', 'utf8', (err, result) => {
+  const first = result;
+  readFile("./content/second.txt", "utf8", (err, result) => {
     if (err) {
-      console.log(err)
-      return
+      console.log(err);
+      return;
     }
-    const second = result
+    const second = result;
     writeFile(
-      './content/result-async.txt',
+      "./content/result-async.txt",
       `Here is the result : ${first}, ${second}`,
-      (err, result) => {
+      (err) => {
         if (err) {
-          console.log(err)
-          return
+          console.log(err);
+          return;
         }
```

Comment:
This might not be part of the assignment, but it would be better to break out the nested writeFiles into separate functions. This will make it easier to read.